app/views/entities/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<header>
+    <div>
+        <%= link_to image_tag('back.png'), group_entities_path(group_id: @group.id), id: "back-icon" %>
+    </div>
+    <h2>
+        Entities
+    </h2>
+    <div>
+        <%= button_to "Logout", destroy_user_session_path, method: :delete %>
+    </div>
+</header>
+
+<%= form_for(Entity.new, url: group_entities_path(group_id: @group.id)) do |f| %>
+    <div class="field mb-4 ">
+        <%= f.text_field :name, placeholder: "Name", autofocus: true, autocomplete: "name", class: "form-control form-control-lg" %>
+    </div>
+
+    <div class="field mb-3">
+        <%= f.number_field :amount, placeholder:"Amount", autofocus: true, autocomplete: "name", class: "form-control form-control-lg" %>
+    </div>
+
+    <div class="field">
+        <%= f.label :group %><br />
+        <%= f.collection_select :group_id, current_user.groups, :id, :name, { class: "options form-control form-control-lg", selected: @group.id } %>
+    </div>
+
+    <div class="actions">
+        <%= f.submit "Add Entity", class: "btn btn-primary btn-lg btn-block splashBtn" %>
+    </div>
+<% end %>
```

Comment:
- [ ] Please style the form input elements and the submit button according to the design guidelines. Please note that the input elements should be wider and take up almost all the available width 👍 
- [ ] Please apply same changes to the form to create a new group on the `groups/new.html.erb` page🙏 
### Design Guideline
![image](https://github.com/Cosmosha/BudgetPal/assets/46839548/3474c1d6-e318-4422-9d13-f2a12c8d684c)

### Your Design
![image](https://github.com/Cosmosha/BudgetPal/assets/46839548/e1c107df-072c-4f34-9e75-1d757a1624f9)